taker-frontend/src/App.tsx

```diff
@@ -159,7 +156,7 @@ export const App = () => {
 
     useEffect(() => {
         const id = "maker-connection-toast";
-        if (!connectedToMaker.online && !toast.isActive(id)) {
+        if (connectedToMakerOrUndefined && !connectedToMakerOrUndefined.online && !toast.isActive(id)) {
```

Comment:
Doesn't this mean, if we start the application and never connect to a maker, that our `connectedToMaker` status is set to `{ online: false }` but we never display the toast?

Is that wanted behavior?

Reply:
yes, because otherwise we get this flickering behavior: 
1. start: `connectedToMakerOrUndefined = undefined `
2. loaded: connectedToMakerOrUndefined = online = true`

Reply:
I think that was expected (and wanted) behavior. Otherwise you would not see the toast if the maker is offline and you open the UI.

Either way, I think given that we have the the maker's status in color on the top, and we show skeletons for price, etc. if we don't have an offer it might not be necessary to show the toast if we *initially* don't have a connection but only show the toast upon losing the connection.

@klochowicz any opinion on that?

Reply:
> I think that was expected (and wanted) behavior. Otherwise you would not see the toast if the maker is offline and you open the UI.

that's wrong. You will see the toast if you are online and the maker is offline. But you won't see the toast if you are offline. 

you're online meaning: your daemon is running or and you have seen the event. 
If you are not online (your daemon is not running) or have not seen the event yet, you can't reason about the maker being here or not.

Reply:
speaking about this: the same is wrong with the online status in the navbar :)